### ISSUE TITLE
Under wish conditions and where in the code is an error like `x is not associated with a coordinate defining true latitude and sharing a subset of dimensions` thrown?

### ISSUE THREAD

**Issue by neumannd** [ioos/compliance-checker#692]
Sorry for mis-using the issue tracker for questions. I wasn't sure whom to contact. Is there a mailing list or something similar?

I get the error

```
§5.6 Horizontal Coorindate Reference Systems, Grid Mappings, Projections
...
* x is not associated with a coordinate defining true latitude and sharing a subset of dimensions
```

In my opinion the tested file is properly structured. Now, I wanted to find out where exactly this error is thrown but I don't find the location in the code. Grepping for `not associated`, `sharing` and/or `subset` did not yield a reasonable result. I only find the test cases `test_cf_integration.py`. I looked briefly through the checks related to §5.6 of the CF Conventions but could not find the appropriate location. Are some tests performed by external libraries?

**Comment by daf** [comment]
Hi @neumannd, I found your string to have existed in a slightly older version of the code, specifically it was changed in this commit: cf2de3b6111cc0f36c5744b9e0c707062de8e5bc which looks to be unreleased so far.  This may be why you are having trouble with the exact wording.  The specific check you're looking for is [check_grid_coordinates](https://github.com/ioos/compliance-checker/blob/b530ab709fbd58ced20d6b3c9989619af98b6282/compliance_checker/cf/cf.py#L2208-L2292).
